cmd/processor/app/processor.go

```diff
@@ -649,3 +654,28 @@ func (p *Processor) setWorkersStatus(triggerInstance trigger.Trigger, status sta
 
 	return nil
 }
+
+// handleSignals creates a signal handler, so on signal processor gracefully terminated
+func (p *Processor) handleSignals() {
+	var captureSignal = make(chan os.Signal, 1)
+	signal.Notify(captureSignal, syscall.SIGTERM, syscall.SIGABRT, syscall.SIGINT)
```

Comment:
Please add a comment on why we are capturing these specific signals - even add the link the the k8s graceful termination doc.